Source/JavaScriptCore/assembler/AssemblerBuffer.h

```diff
@@ -460,7 +512,7 @@ namespace JSC {
 #if CPU(ARM64)
             static_assert(sizeof(value) == 4);
 #if ENABLE(JIT_CHECKSUM_ASSEMBLER_BUFFER)
```

Comment:
I don't see `JIT_CHECKSUM_ASSEMBLER_BUFFER` enabled for x64.  How is it used?

Also, have you checked if the x64 code flow will actually take this path that uses the `BufferChecksum`?

Reply:
Good catch: I accidentally removed the chunk which did before submitting. I had checked (i.e. compiled+run on x86 and set a breakpoint on the appropriate function) but it must have been with the modified revision.

Reply:
Fixed, it's now enabled on all platforms:

```
#if !defined(ENABLE_JIT_CHECKSUM_ASSEMBLER_BUFFER)
 #define ENABLE_JIT_CHECKSUM_ASSEMBLER_BUFFER 1
 #endif
 ```